dhctl/pkg/operations/bootstrap/steps.go

```diff
@@ -283,8 +283,18 @@ func SetupSSHTunnelToRegistryPackagesProxy(ctx context.Context, sshCl node.SSHCl
 	tun := sshCl.ReverseTunnel(fmt.Sprintf("%s:%s:%s:%s", listenAddress, port, listenAddress, port))
 	err = tun.Up()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf(`Cannot setup tunnel to control-plane host: %w.
```

Comment:
We special preflight check for it https://github.com/deckhouse/deckhouse/blob/main/dhctl/pkg/preflight/ssh.go#L37. Need to fix it there